tests/testsuite/cargo_remove/update_lock_file/mod.rs

```diff
@@ -0,0 +1,25 @@
+use cargo_test_support::compare::assert_ui;
```

Comment:
What does it mean here by `update_lock_file`? I may miss something but I cannot see any difference between this test case and remove_basic.

Reply:
Huh, I think [the cargo-edit version](https://github.com/killercup/cargo-edit/pull/780/files#diff-de50f50ee99d34880271073ffb429e7a0119d2089120776e374b469b87d8a319) has an `in/Cargo.lock` and `out/Cargo.lock`.  Looks like those were lost in translation

@cassaundra could you double check if anything else is missing?

Reply:
Ah yes, I forgot to force add these files. Fixed now.